cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -73,6 +74,7 @@ data CardanoTestnetOptions = CardanoTestnetOptions
   , cardanoNodeLoggingFormat :: NodeLoggingFormat
   , cardanoNumDReps :: NumDReps -- ^ The number of DReps to generate at creation
   , cardanoEnableNewEpochStateLogging :: Bool -- ^ if epoch state logging is enabled
+  , cardanoConfigFilesBehaviour :: ConfigFilesBehaviour -- ^ Whether to continue tests after generating config files.
```

Comment:
I don think this is necessary. The default behaviour of `cardano-testnet` is to generate some configuration and then create a testnet based on that config. This should really exist as separate functionality, it seems a bit shoehorned here.